src/store/Network.ts

```diff
@@ -302,7 +302,8 @@ export default {
                 return;
             } else {
                 const nodeService = new NodeService();
-                let nodesList = await nodeService.loadNodesFromStatisticService(networkType);
+                const statisticsServiceNodes = !isOffline ? await nodeService.loadNodesFromStatisticService(networkType) : undefined;
```

Comment:
you might consider keeping this consistent with your other naming - `loadNodesFromStatisticsService`.